Ginger/Ginger/AnalyzerLib/AnalyzerPage.xaml.cs

```diff
@@ -213,6 +213,14 @@ private async Task Analyze()
 
             finally
             {
+                foreach (var issue in mIssues)
+                {
+                    if (issue.CanAutoFix == AnalyzerItemBase.eCanFix.Yes && issue.FixItHandler != null && mAnalyzerUtils.SelfHealingAutoFixIssue)
+                    {
+                        issue.FixItHandler.Invoke(issue, null);
```

Comment:
need to use for loop. And if the issue canAutoFix and FixItHandler is called, should be removed from mIssues. 

Otherwise the issue will be fixed but still shown on the Analyzer UI if running the flow